### PR TITLE
FieldConfiguration extension to exclude a list of Enums from display

### PR DESCRIPTION
**Pull request by MattDavies**

### CODE REVIEW

**Comment by robdmoore** [comment]
Build failed

File: ChameleonForms/Component/Config/FieldConfiguration.cs [hunk 1]
```diff
@@ -379,6 +379,18 @@ public interface IFieldConfiguration : IHtmlString
         /// </summary>
         /// <returns>A readonly field configuration</returns>
         IReadonlyFieldConfiguration ToReadonly();
+
+        /// <summary>
+        /// Excludes one or more Enum values from the generated field.
+        /// </summary>
+        /// <param name="enumValues">The value of Enum(s) to exclude from the generated field.</param>
+        /// <returns></returns>
+        IFieldConfiguration ExcludeValues(params Enum[] enumValues);
```
Comment (robdmoore) [inline ChameleonForms/Component/Config/FieldConfiguration.cs]: Pity we can't have type inference.

Do you think ExcludeValues is discoverable enough?

**Comment by robdmoore** [comment]
https://trello.com/c/PmCdrInS/29-exclude-specific-enum-values-from-being-selected-on-the-model-side-including-validation

**Comment by MattDavies** [comment]
I split the trello card into two earlier, one to cover client side and the other to cover model/validation.

File: ChameleonForms/Component/Config/FieldConfiguration.cs (hunk 1 above)
Comment (MattDavies) [inline ChameleonForms/Component/Config/FieldConfiguration.cs]: I'm open to suggestions if you can think of something nicer.

**Comment by robdmoore** [comment]
LYW

**Comment by robdmoore** [comment]
I've updated the doco: https://github.com/MRCollective/ChameleonForms/wiki/enum

**Comment by robdmoore** [comment]
Released as v1.2.0

**Pull request merged**